src/test/java/com/easyrest/tests/owner/OwnerSignIn.java

```diff
@@ -0,0 +1,33 @@
+package com.easyrest.tests.owner;
+
+import com.easyrest.config.ConfigProvider;
+import com.easyrest.facade.SignInFacade;
+import com.easyrest.pages.OwnerPanel;
+import com.easyrest.pages.SignInPage;
+import com.easyrest.tests.BaseTest;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class OwnerSignIn extends BaseTest {
+
+    private final String ownerEmail = ConfigProvider.ownerEmail;
+    private final String ownerPassword = ConfigProvider.ownerPassword;
+    private SignInPage signInPage;
+    private SignInFacade signInFacade;
+    private OwnerPanel ownerPanel;
+
+    @Test
+    public void signInTestsOwner() {
+        signInPage = new SignInPage(driver);
+        signInFacade = new SignInFacade(driver);
+        ownerPanel = new OwnerPanel(driver);
+        signInPage.goToSignInPage();
+        signInFacade.signIn(ownerEmail, ownerPassword);
+        ownerPanel.waitForPagePresence();
+        Assert.assertEquals(driver.getCurrentUrl(), ConfigProvider.ownerPageUrl, "Failed to login");
+    }
+}
```

Comment:
Can not run this test, compile error
java: cannot find symbol
  symbol:   class BaseComponent
  location: package com.easyrest.components

Reply:
in the 35th pull request, I deleted this line, update your main

Reply:
So why did you remove that line 12 minutes ago if you did it in the 35th pull request?

Reply:
I don't know why they reappeared

Reply:
@vadymtsehelny @Zeicks, what is the current status of that issue?

Reply:
Resolve